crates/trigger/src/cli.rs

```diff
@@ -175,14 +166,10 @@ where
         match &self.log {
             Some(l) => Some(l),
             None => {
-                if !self.from_registry {
-                    if let Ok(dir) = std::env::var(SPIN_STATE_DIR) {
-                        let s = PathBuf::from(dir).join("logs");
-                        return Some(s);
-                    } else {
-                        tracing::info!("Unable to retrieve SPIN_STATE_DIR environment variable to persist logs",);
```

Comment:
@michelleN FYI I felt OK removing this because the logic will change significantly with https://github.com/fermyon/spin/issues/1175